### DISCUSSION TITLE
plugins not available?

### DISCUSSION THREAD

**Discussion by disco-io** [lowlighter/metrics#1152] [Q&A]
How can I add the Anilist and Music stuff to my profile? There seems to be a problem with the plugins not being available on my web instance :c

_***cool project btw_

![image](https://user-images.githubusercontent.com/80224791/180663187-e244580f-623d-401a-b895-d271e2ce15ec.png)

**Comment by jayvdb** [comment]
Per https://github.com/lowlighter/metrics/tree/master/source/plugins/anilist and https://github.com/lowlighter/metrics/tree/master/source/plugins/music

> Web instances must configure `settings.json`: `metrics.run.puppeteer.scrapping`

More info about that setting at https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/documentation/setup/web.md#2%EF%B8%8F3%EF%B8%8F-extra-features-configuration

**Reply by lowlighter** [reply]
As @jayvdb if you host your own web instance and switched to v3.26, you'll need to configure permissions in `settings.json` (see [3.26 patchnote](https://github.com/lowlighter/metrics/releases/tag/v3.26) for more informations about it)

It may be a bit misleading since it's still the same major version, but I didn't want to upgrade to v4 since it doesn't affect the GitHub Action (which I feel like is the primary way of using this app). These changes were made to make it easier to manage CPU/API/Network intensive tasks and to mitigate possible security vulnerabilities from unmaintained dependencies.

____

If you're talking about the shared web instance ([metrics.lecoq.io](https://metrics.lecoq.io)), then these plugins are not enabled, as Anilist have a rate limit which isn't suitable for many users and the Music plugin makes heavy use of puppeteer (which spawns a chromium tab to scrap stuff and thus time/cpu consuming)

___

Hope it makes it clearer 😅
